java/ingest/src/main/java/sleeper/ingest/IngestRecords.java

```diff
@@ -64,4 +69,8 @@ public long close() throws IOException, IteratorException, InterruptedException,
                 .mapToLong(FileInfo::getNumberOfRecords)
                 .sum();
     }
+
+    public List<FileInfo> closeReturningFileInfoList() throws IOException, IteratorException, StateStoreException {
+        return ingestCoordinator.closeReturningFileInfoList();
+    }
```

Comment:
It would probably be neater to just have one close method. We could have an IngestResult class that wraps the list so we have a neater way to inspect the results?

If we want to retain the original close method we could chain them together to make it easier to cover both with tests.